packages/network-contracts/test/hardhat/OperatorTokenomics/Operator.test.ts

```diff
@@ -182,6 +313,27 @@ describe("Operator contract", (): void => {
             expect(formatEther(contractBalanceAfterDelegate)).to.equal("1000.0")
             expect(formatEther(contractBalanceAfterUndelegate)).to.equal("0.0")
         })
+
+        it("allows delegate via transferAndCall by passing a bytes32 data param", async function(): Promise<void> {
+            const { token } = sharedContracts
+            await setTokens(delegator, "1000")
+            const operator = await deployOperator(operatorWallet)
+            // assume the address was encoded by converting address -> uint256 -> bytes32 -> bytes
+            const data = hexZeroPad(delegator.address, 32)
+            await (await token.connect(delegator).approve(operator.address, parseEther("1000"))).wait()
+            await expect(token.connect(delegator).transferAndCall(operator.address, parseEther("1000"), data))
+                .to.emit(operator, "Delegated").withArgs(delegator.address, parseEther("1000"))
+        })
+    
+        it("allows delegate without delegation policy being set", async function(): Promise<void> {
+            const { token } = sharedContracts
+            await setTokens(delegator, "1000")
+            const operator = await deployOperator(operatorWallet)
+            await (await operator.connect(operatorWallet).setDelegationPolicy(hardhatEthers.constants.AddressZero, 0)).wait()
```

Comment:
```suggestion
            const operator = await deployOperatorContract(operatorWallet, {
                delegationPolicyAddress: hardhatEthers.constants.AddressZero
            })
```
and add overrides to deployOperator and deployOperatorContract

Reply:
Updated